conan/musl/v1.1.18/conanfile.py

```diff
@@ -28,13 +21,9 @@ def build_requirements(self):
 
     def source(self):
         git = tools.Git(folder="musl")
-        git.clone("git://git.musl-libc.org/musl/",branch=self.version)
+        git.clone("https://github.com/fwsGonzo/musl.git",branch="set_thread")
```

Comment:
Are we merging that branch to a tag ? @staiyeba  will have to pick these changes up and merge the changes i made to syscalls.h for aarch64 to make it work i think.